docs/user-guide/multiple_sources.md

```diff
@@ -0,0 +1,62 @@
+# Multiple Sources for an Application
```

Comment:
What happens in the sync processes: from repos, to destination?
Any new intermediate status if one source's change get synced and not the other one?
What happens in terms of notification, health checks?
I assumed the granularity remains the "app", but just to make sure.

Reply:
We do not have an intermediate status. One anyone of the source is unable to get synced, it will mark the app as Unhealthy.

Reply:
ok. I dont consider it a problem tbh. Maybe just a line in the doc to avoid wrong expectations. Wdyt?